makefile.shade

```diff
@@ -82,7 +82,7 @@ var buildTarget = "compile"
 
 #change-default-build-target
     @{
-        buildTarget = "verify";
+        buildTarget = "--parallel verify";
```

Comment:
Need a way to disable `--parallel` if enabling it in this fashion. This also isn't a great default due to the spew of messages coming from the different threads -- makes finding a problem more difficult. (Could change the `Parallel.ForEach()` blocks to spew with `msbuild`-like thread ids but that's probably too much work for this private build system.)